packages/gcom/package.py

```diff
@@ -9,17 +9,20 @@ class Gcom(Package):
     homepage = "https://code.metoffice.gov.uk/trac/gcom"
     svn = "file:///g/data/ki32/mosrs/gcom/main/trunk"
 
-    maintainers = ["scottwales"]
+    maintainers("scottwales", "paulleopardi")
 
     # See 'fcm kp fcm:gcom.xm' for release versions
     version("7.8", revision=1147)
     version("7.9", revision=1166)
     version("8.0", revision=1181)
+    version("8.1", revision=1215)
+    version("8.2", revision=1251)
+    version("8.3", revision=1288)
 
     variant("mpi", default=True, description="Build with MPI")
 
     depends_on("fcm", type="build")
-    depends_on("mpi", when="+mpi")
+    depends_on("openmpi@4.1.3:", when="+mpi", type=("build", "link", "run"))
```

Comment:
Please add a reference to the document.
Also, is it possible to do `depends_on("openmpi@4.1.3:", when="+mpi ^openmpi" ...`